source/widgets/onboarding_setup/wizard_pages.py

```diff
@@ -0,0 +1,345 @@
+                typical_paths = [
+                    Path("~/Desktop/Blender Launcher V2").expanduser(),
+                    Path("~/OneDrive/Desktop/Blender Launcher V2").expanduser(),
+                ]
+                exceptions = []
+                for pth in typical_paths:
+                    try:
+                        generate_program_shortcut(
+                            pth,
+                            exe=str(self.prop_settings.exe_location),
+                        )
+                        break
+                    except Exception as e:
+                        exceptions.append(e)
+                if len(exceptions) == len(typical_paths):  # all paths failed to generate
+                    raise Exception("Exceptions raised while generating desktop shortcuts: {exceptions}")
+
+
+TITLEBAR_LABEL_TEXT = """This disables the custom title bar and uses the OS's default titlebar.
+
+In Linux Wayland environments, this is recommended because you will be
+able to use the title for moving and resizing the windows.
+Our main method of moving and resizing works best on X11."""
+
+HIGH_DPI_TEXT = """This enables high DPI scaling for the program.
+automatically scales the user interface based on the monitor's pixel density."""
+
+
+class AppearancePage(BasicOnboardingPage):
+    def __init__(self, prop_settings: PropogatedSettings, parent: BlenderLauncher):
+        super().__init__(prop_settings, parent=parent)
+        self.setTitle("BLV2 appearance")
+        self.setSubTitle("Configure how BLV2 Looks")
+        self.layout_ = QVBoxLayout(self)
+
+        self.titlebar = QCheckBox("Use System Titlebar", self)
+        self.titlebar.setChecked(get_use_system_titlebar())
+        titlebar_label = QLabel(TITLEBAR_LABEL_TEXT, self)
+        self.highdpiscaling = QCheckBox("High DPI Scaling")
+        self.highdpiscaling.setChecked(get_enable_high_dpi_scaling())
+        highdpiscaling_label = QLabel(HIGH_DPI_TEXT)
+
+        self.layout_.addWidget(titlebar_label)
+        self.layout_.addWidget(self.titlebar)
+        self.layout_.addWidget(highdpiscaling_label)
+        self.layout_.addWidget(self.highdpiscaling)
+
+    def evaluate(self):
+        set_use_system_titlebar(self.titlebar.isChecked())
+        set_enable_high_dpi_scaling(self.highdpiscaling.isChecked())
+
+
+BACKGROUND_SUBTITLE = """BLV2 can be kept alive in the background with a system tray icon.\
+ This can be useful for reading efficiency and other features, but it is not totally necessary."""
+
+
+class BackgroundRunningPage(BasicOnboardingPage):
+    def __init__(self, prop_settings: PropogatedSettings, parent: BlenderLauncher):
+        super().__init__(prop_settings, parent=parent)
+        self.setTitle("Running BLV2 in the background")
+        self.setSubTitle(BACKGROUND_SUBTITLE)
+        self.layout_ = QVBoxLayout(self)
+
+        self.enable_btn = QCheckBox("Run BLV2 in the background (Minimise to tray)")
```

Comment:
`Blender Launcher` instead of `BLV2`
```suggestion
        self.setTitle("Running Blender Launcher in the background")
        self.setSubTitle(BACKGROUND_SUBTITLE)
        self.layout_ = QVBoxLayout(self)

        self.enable_btn = QCheckBox("Run Blender Launcher in the background (Minimise to tray)")
```